bochs/iodev/usb/scsi_device.cc

```diff
@@ -485,12 +485,12 @@ Bit8u* scsi_device_t::scsi_get_buf(Bit32u tag)
   return r->dma_buf;
 }
 
-Bit32s scsi_device_t::scsi_send_command(Bit32u tag, Bit8u *buf, int lun, bool async)
+Bit32s scsi_device_t::scsi_send_command(Bit32u tag, Bit8u *buf, const Bit8u cmd_len, int lun, bool async)
```

Comment:
why would you add const here ?
it doesn't matter as cmd_len is copied by value anyway

Reply:
Sorry, habit I guess. Wanted to make sure that I didn't modify it throughout. You may remove it if you wish.